README.md

```diff
@@ -15,6 +15,21 @@ def deps do
 end
 ```
 
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/bepaid](https://hexdocs.pm/bepaid).
+## Configuration
```

Comment:
@MikeAndrianov once you figure out how to run tests in parent app, will need to provide instruction how to setup test env with `bepaid_ex` library.